docs/ROLLDOWN.md

```
@@ -0,0 +1,47 @@
+{
+  "vite": "npm:rolldown-vite@7.2.5"
+}
+```
+
+Версия **зафиксирована** (`7.2.5`) для предотвращения неожиданных breaking changes.
+
+## Используемые плагины
+
+| Плагин | Версия | Совместимость |
+|--------|--------|---------------|
+| `@vitejs/plugin-vue` | ^6.0.1 | ✅ Совместим (официальный плагин, приоритет команды Vite) |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)